build.gradle.kts

```diff
@@ -22,4 +22,4 @@ tasks.register("preMerge") {
     dependsOn(":examples:android-gradle:check")
     dependsOn(":examples:android-gradle-kts:check")
     dependsOn(gradle.includedBuild("plugin-build").task(":check"))
-}
+}
```

Comment:
I propose we add a `.editorconfig` defining an empty line should always exist in all files.
This helps with diffing where avoids two lines as modified (appending `\n` at the end of the previously last line). 

i.e:

```
[*]
indent_style = space
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false
```

Reply:
Yup done here: https://github.com/getsentry/sentry-android-gradle-plugin/pull/54